_data/es/settings.yml

```diff
@@ -57,6 +57,7 @@ contact_page:
       'HHS Grants.gov': Grants.gov del HHS
       'HHS National Institutes of Health (NIH)': Institutos Nacionales de la Salud
       'Housing and Urban Development': Departamento de Vivienda y Desarrollo Urbano de EE. UU.
+      'Servicio de Impuestos Internos (IRS)' : Internal Revenue Service (IRS)
```

Comment:
- Is the key / value supposed to be flipped here? Other agencies use English as the key and the translated version as the value.
- Nit: Excess space around the colon

```suggestion
      'Internal Revenue Service (IRS)': Servicio de Impuestos Internos (IRS)
```